packages/gafl-webapp-service/src/handlers/analytics-handler.js

```diff
@@ -1,11 +1,31 @@
 import { ANALYTICS } from '../constants.js'
+import db from 'debug'
+const debug = db('webapp:analytics-handler')
+
 /**
  * Analytics route handler
  * @param request
  * @param h
  * @returns {Promise}
  */
 
+export const trackGTM = async request => {
+  const pageOmit = await pageOmitted(request)
+  const canTrack = await trackAnalyticsAccepted(request, pageOmit)
+  const optDebug = process.env.ENABLE_ANALYTICS_OPT_IN_DEBUGGING?.toLowerCase() === 'true'
+  const gtmContainerId = process.env.GTM_CONTAINER_ID
+  if (optDebug && gtmContainerId) {
+    if (canTrack === true) {
+      debug(`GTM Container Id: ${gtmContainerId} is being tracked`)
+    } else if (pageOmit === true) {
+      debug(`GTM Container Id: ${gtmContainerId} is not being tracked for current page`)
+    } else {
+      debug(`GTM Container Id: ${gtmContainerId} is not being tracked`)
```

Comment:
The logging here doesn't quite match up with what we had previously – it used to be that we would log if the session was being tracked, and that's what it looks like you are still intending from the tests. However this will just output the same GTM container ID each time, as that is a set thing for the environment.